frontend/src/Home/Page.js

```diff
@@ -23,6 +23,13 @@ const ButtonContainer = styled.div`
   width: 250px;
 `
 
+const LoginContainer = styled.div`
+  justify-content: center;
+  min-height: 75vh;
+  display: flex;
+  flex-direction: column;
```

Comment:
This will definitively give you the vertical layout you need for the login area, but it's not horizontally centered correctly. You'll probably want to add a separate div or something else around the login container to center it.

Reply:
I might just be pedantic about it, so don't work super hard on it if you can't get it centered

Reply:
fixed centering in future commit